### DISCUSSION TITLE
Is it possible to display the expert mode icon in full screen?

### DISCUSSION THREAD

**Discussion by REGoman** [OnionUI/Onion#1347] [Q&A]
If there is a "bg-ra-list-item" file in the theme, the background will be displayed but the icon will be cut off.
![sss](https://github.com/OnionUI/Onion/assets/122949549/c4d024cd-8612-4708-b9c1-ffb87e9238d8)
The icon is cut off like this. Can't I make it full screen?

**Comment by Aemiii91** [comment]
Not possible. Please note that the frontend is made by Miyoo and is closed source.

**Reply by REGoman** [reply]
thank you!
